src/shared/config.rs

```diff
@@ -11,6 +11,8 @@ pub const DEFAULT_CONFIG_PATH: &str = "~/.config/spotifatius/config.toml";
 pub struct Config {
     #[serde(default = "default_polybar_config")]
     pub polybar: PolybarConfig,
+    #[serde(default = "default_format")]
+    pub format: String,
```

Comment:
What do you think about having some extra formatting options?

```rust
pub struct TemplateConfigs {
	#[serde(default = "default_text_template")]
	pub text: TemplateConfig,
	#[serde(default = "default_tooltip_template")]
	pub tooltip: TemplateConfig,
}

pub struct TemplateConfig {
	pub default: String,
	pub playing: String,
	pub stopped: String,
	pub paused: String,
	pub added: String,
	pub removed: String,
}
```

This would allow configuration of icons and more so we do not hardcode those 2 unicode characters.